configure.ml

```diff
@@ -988,6 +1002,9 @@ let esc s = if String.contains s ' ' then "\"" ^ s ^ "\"" else s
 
 (** * Summary of the configuration *)
 
+let pr_native = function
+  | NativeYes -> "no" | NativeNo -> "no" | NativeOndemand -> "ondemand"
```

Comment:
This seems buggy.

Reply:
Oups, indeed. That was just printing in the configure script though. Fixed.